test/integration-test.sh

```diff
@@ -5,18 +5,34 @@ DIR=$PWD
 go build ./cmd/stupid
 curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash
 
-cd ./fabric-samples/test-network
-echo y |  ./network.sh down
-echo y |  ./network.sh up createChannel
-cp -r organizations "$DIR"
+cd ./fabric-samples/
+if [ $1 == "1.4" ]; then
```

Comment:
`switch case` or `if...elif` ?

Reply:
personal prefer switch here, when 1.4 from tls to archived, we can remove the switch block without any further adjust.